src/applications/static-pages/health-care-manage-benefits/get-medical-records-page/components/App/index.js

```diff
@@ -9,8 +9,9 @@ import UnauthContent from '../UnauthContent';
 import { isCernerLive } from 'platform/utilities/cerner';
 import { selectIsCernerPatient } from 'platform/user/selectors';
 
-export const App = ({ isCernerPatient }) => {
-  if (!isCernerLive) {
+export const App = ({ isCernerPatient, showNewGetMedicalRecordsPage }) => {
+  // Show legacy content if Cerner isn't live or if we explicitly shouldn't show the page via a feature flag.
+  if (!isCernerLive || showNewGetMedicalRecordsPage === false) {
```

Comment:
It would be cool if these flags were consolidated...like if we set `showNewGetMedicalRecordsPage` to true via Flipper then it should be safe to say Cerner must be live. What do you think?

Reply:
After Friday `isCernerLive` will be omitted from the codebase I believe 🙂